library/functions.po

```diff
@@ -379,10 +373,10 @@ msgid ""
 "It cannot be subclassed further.  Its only instances are ``False`` and "
 "``True`` (see :ref:`bltin-boolean-values`)."
 msgstr ""
-"返回一個布林值，``True`` 或者 ``False``。 *x* 使用標準的 :ref:`真值測試過程 "
-"<truth>` 來轉換。如果 *x* 是假的或者被省略，返回 ``False``；其他情況返回 "
-"``True``。:class:`bool` 類是 :class:`int` 的子類（參見 :ref:"
-"`typesnumeric`）。其他類不能繼承自它。它只有 ``False`` 和 ``True`` 兩個例項"
+"返回一個布林值，即 ``True`` 或者 ``False``。 *x* 使用標準的 :ref:`真值測試過"
```

Comment:
There's a redundant space after the period.

Also, as the ref here is self-containing the display text here which is in Chinese, we should trim the space before and after the ref, you should use a ["separator without a width"](https://github.com/python/python-docs-zh-tw#rst) here.

Reply:
Thanks! I think I really know the meaning of the "separator without a width" in the translation tutorial 👍